src/crypto/Address.dto.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import {
+  Address,
+  ByronAddress,
+  BaseAddress,
+  PointerAddress,
```

Comment:
Heads up: `PointerAddress` and `EnterpriseAddress` are not implemented in our native bindings.

Reply:
Roger that, commenting for now.